.env.lacchain

```diff
@@ -43,10 +43,21 @@ HAPI_SYNC_PRODUCER_INFO_INTERVAL=1
 HAPI_SYNC_PRODUCER_CPU_INTERVAL=6
 HAPI_SYNC_STATS_INTERVAL=60
 HAPI_SYNC_SCHEDULE_HISTORY_INTERVAL=86400
+HAPI_EOS_GET_FAUCET_ACCOUNT=
+HAPI_EOS_FAUCET_ACCOUNT=
+HAPI_EOS_FAUCET_ACCOUNT_PASSWORD=
+GOOGLE_APPLICATION_CREDENTIALS=
+HAPI_RE_CAPTCHA_PROJECT_ID=
+HAPI_PUBLIC_RE_CAPTCHA_KEY=
+HAPI_CREATE_ACCOUNT_ACTION_NAME=
+HAPI_SYNC_EXCHANGE_RATE=
+HAPI_EOS_EXCHANGE_RATE_API=
+HAPI_COINGECKO_API_TOKEN_ID=
+HAPI_REWARDS_TOKEN=
 
 #webapp
 PORT=3000
-REACT_APP_TAG=dev
+REACT_APP_VERSION=dev
 REACT_APP_NAME=$npm_package_name
 REACT_APP_TITLE=LACChain Network Dashboard
 REACT_APP_DEFAULT_PRODUCER_LOGO=https://ipfs.eosio.cr/ipfs/QmS5XzgSEAubehxZVwiHnaLpo3Pv2E5yeEPBhhv6JWSKHE
```

Comment:
no longer using IPFS for this , use this URL instead

https://eosio.online/images/lacchain.jpg